homeassistant/components/romy/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""ROMY Integration."""
+
+import romy
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, PLATFORMS
+from .coordinator import RomyVacuumCoordinator
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Initialize the ROMY platform via config entry."""
+
+    password = config_entry.data.get(CONF_PASSWORD, "")
```

Comment:
The schema requires a password, therefore you can access it like:
```suggestion
    password = config_entry.data[CONF_PASSWORD]
```

Reply:
Unfortunate i have two schema, one with and one without password, so it could be the case its empty